### PR TITLE
[C#] bump: dotnet to 1.9.0

### PR DESCRIPTION
**Pull request by singhk97**
## Linked issues

closes: #minor

## Details
- https://github.com/microsoft/teams-ai/pull/2210
- https://github.com/microsoft/teams-ai/pull/2214
- https://github.com/microsoft/teams-ai/pull/2227

### CODE REVIEW

**Comment by singhk97** [comment]
We should merge this once the package is deployed to nuget.

File: js/yarn.lock [hunk 1]
```diff
@@ -824,6 +824,11 @@
   resolved "https://registry.yarnpkg.com/@opentelemetry/semantic-conventions/-/semantic-conventions-1.28.0.tgz#337fb2bca0453d0726696e745f50064411f646d6"
   integrity sha512-lp4qAiMTD4sNWW4DbKLBkfiMZ4jbAboJIGOQr5DvciMRI494OapieI9qiODpOt0XBr1LjIDy1xAGAnVs5supTA==
 
+"@opentelemetry/semantic-conventions@^1.19.0":
```
Comment (corinagum) [inline js/yarn.lock:827]: Can we remove this change from the PR?

**Review by corinagum — CHANGES_REQUESTED** [review]
Just one nit

File: js/yarn.lock (hunk 1 above)
Comment (singhk97) [reply js/yarn.lock]: thanks for calling it out. Idk how the js changes got in here

**Review by corinagum — APPROVED** [review]

**Pull request merged**